docs/docs/getting-started.server.mdx

```diff
@@ -573,40 +573,68 @@ on how to use MDX with Gatsby.
   <summary>Expand example</summary>
 
   ```js path="next.config.js"
-  module.exports = {
-    // Prefer loading of ES Modules over CommonJS
-    experimental: {esmExternals: true},
+  import nextMDX from '@next/mdx'
+
+  const withMDX = nextMDX({
+    // By default only the .mdx extension is supported.
+    extension: /\.mdx?$/,
+    options: {/* providerImportSource: …, otherOptions… */}
+  })
+
+  export default withMDX({
```

Comment:
I believe Next itself, on pages, asks to first assign to a const, and the export default, so that there’s a name attached.
Should we do the same, or is it fine?

Reply:
I don’t really understand what you mean, but this syntax is very similar to [this paragraph on using MDX](https://nextjs.org/docs/advanced-features/using-mdx#setup-nextmdx-in-nextjs).

Reply:
I believe there’s a warning in Next.js for:

```js
export default () => {}
```

It wants you to assign to a name:

```js
const SomePage = () => {}
export default SomePage
```

That is to attach a name. I think that’s good here too.

Reply:
This isn’t a page, it’s the configuration file (`next.config.js`).

Reply:
I understand it is not a page.
I use Next.js recommendation as an example.
It is still good to name things.
My assumption is that errors will occur and a name will improve stack traces.